### PR TITLE
Updates microservices eBook .NET 6.0

### PR DESCRIPTION
**Pull request by sughosneo**
This PR contains all the changes which updates `.NET Microservices: Architecture for Containerized .NET Applications` eBook to `.NET 6.0`

### CODE REVIEW

File: docs/architecture/microservices/docker-application-development-process/index.md [hunk 1]
```diff
@@ -13,9 +13,9 @@ ms.date: 02/02/2021
...
+**Visual Studio (for Windows).** Docker-based .NET 6 application development with Visual Studio requires Visual Studio 2022 version 17.0 or later. Visual Studio 2022 comes with tools for Docker already built in. The tools for Docker let you develop, run, and validate your applications directly in the target Docker environment. You can press F5 to run and debug your application (single container or multiple containers) directly into a Docker host, or press CTRL+F5 to edit and refresh your application without having to rebuild the container. This IDE is the most powerful development choice for Docker-based apps.
```
Comment (IEvangelist) [inline docs/architecture/microservices/docker-application-development-process/index.md:16]:
```suggestion
**Visual Studio (for Windows).** Docker-based .NET 6 application development with Visual Studio requires Visual Studio 2022 version 17.0 or later. Visual Studio 2022 comes with tools for Docker already built in. The tools for Docker let you develop, run, and validate your applications directly in the target Docker environment. You can press <kbd>F5</kbd> to run and debug your application (single container or multiple containers) directly into a Docker host, or press <kbd>CTRL</kbd> + <kbd>F5</kbd> to edit and refresh your application without having to rebuild the container. This IDE is the most powerful development choice for Docker-based apps.
```

File: docs/architecture/microservices/index.md [hunk 2]
```diff
@@ -43,7 +43,7 @@ After you have studied this guide, your next step would be to learn about produc
 
 ## Version
 
-This guide has been revised to cover **.NET 5** version along with many additional updates related to the same “wave” of technologies (that is, Azure and additional third-party technologies) coinciding in time with the .NET 5 release. That’s why the book version has also been updated to version **5.0**.
+This guide has been revised to cover **.NET 6** version along with many additional updates related to the same “wave” of technologies (that is, Azure and additional third-party technologies) coinciding in time with the .NET 6 release. That’s why the book version has also been updated to version **6.0**.
```
Comment (IEvangelist) [inline docs/architecture/microservices/index.md:46]:
```suggestion
This guide has been revised to cover **.NET 6** version along with many additional updates related to the same "wave" of technologies (that is, Azure and additional third-party technologies) coinciding in time with the .NET 6 release. That’s why the book version has also been updated to version **6.0**.
```

File: docs/architecture/microservices/net-core-net-framework-containers/index.md [hunk 3]
```diff
@@ -1,11 +1,11 @@
 ---
-title: Choosing Between .NET 5 and .NET Framework for Docker Containers
-description: .NET Microservices Architecture for Containerized .NET Applications | Choosing Between .NET 5 and .NET Framework for Docker Containers
+title: Choosing Between .NET 6 and .NET Framework for Docker Containers
+description: .NET Microservices Architecture for Containerized .NET Applications | Choosing Between .NET 6 and .NET Framework for Docker Containers
 ms.date: 02/02/2021
```
Comment (IEvangelist) [inline docs/architecture/microservices/net-core-net-framework-containers/index.md:4]:
```suggestion
ms.date: 11/19/2021
```

File: docs/architecture/microservices/net-core-net-framework-containers/net-framework-container-scenarios.md [hunk 4]
```diff
@@ -1,60 +1,64 @@
...
   [https://docs.microsoft.com/dotnet/core/docker/introduction](../../../core/docker/introduction.md)
 
 - **.NET Components Overview** \
   [https://docs.microsoft.com/dotnet/standard/components](../../../standard/components.md)
-
+  
 >[!div class="step-by-step"]
 >[Previous](net-core-container-scenarios.md)
->[Next](container-framework-choice-factors.md)
+>[Next](container-framework-choice-factors.md)
```
Comment (IEvangelist) [inline docs/architecture/microservices/net-core-net-framework-containers/net-framework-container-scenarios.md:64]:
```suggestion
>[Next](container-framework-choice-factors.md)

```

**Review by IEvangelist — CHANGES_REQUESTED** [review]
This looks great, thank you - just a few minor nits before we can :shipit:

**Comment by sughosneo** [comment]
> This looks great, thank you - just a few minor nits before we can :shipit:

Thank you @IEvangelist for your review.

File: docs/architecture/microservices/docker-application-development-process/docker-app-development-workflow.md [hunk 5]
```diff
@@ -47,21 +47,21 @@ To begin, make sure you have [Docker Community Edition (CE)](https://docs.docker
 
 [Get started with Docker CE for Windows](https://docs.docker.com/docker-for-windows/)
 
-In addition, you need Visual Studio 2019 version 16.8, with the **.NET Core cross-platform development** workload installed, as shown in Figure 5-2.
+In addition, you need Visual Studio 2022 version 17.0, with the **.ASP.NET and web development** workload installed, as shown in Figure 5-2.
```
Comment (nishanil) [inline docs/architecture/microservices/docker-application-development-process/docker-app-development-workflow.md:50]: In the earlier line the link has the title "Docker CE for Windows". Can you change that to Docker Desktop for Windows?

**Comment by nishanil** [comment]
@sughosneo  I noticed that this file: docker-application-development-process/docker-app-development-workflow.md still has references to Visual Studio 2019. Either drop 2019 or update to VS 2022. Also, update screenshots to latest.

File: docs/architecture/microservices/net-core-net-framework-containers/net-framework-container-scenarios.md [hunk 6]
```diff
@@ -1,60 +1,64 @@
...
+
+.NET Core is a revolutionary step forward from .NET Framework. It offers a host of advantages over .NET Framework across the board from productivity to performance, from cross-platform support to developer satisfaction. If you are using .NET Framework and planning to migrate your application to .NET Core and later to .NET 6, then refer to this guide [Porting Existing ASP.NET Apps to .NET Core](https://docs.microsoft.com/dotnet/architecture/porting-existing-aspnet-apps/).
 
 ### Additional resources
 
 - **.NET fundamentals** \
   [https://docs.microsoft.com/dotnet/fundamentals](../../../fundamentals/index.yml)
 
-- **Porting Projects to .NET 5** \
+- **Porting Projects to .NET 6** \
```
Comment (nishanil) [inline docs/architecture/microservices/net-core-net-framework-containers/net-framework-container-scenarios.md:53]: The video you are referencing is still .NET 5?

**Review by nishanil — COMMENTED** [review]
@sughosneo Looks good. Some minor things to be taken care of.

File: docs/architecture/microservices/net-core-net-framework-containers/net-framework-container-scenarios.md [hunk 7]
```diff
@@ -1,60 +1,64 @@
...
 
-## Using a platform or API that doesn't support .NET 5
+## Using a platform or API that doesn't support .NET 6
 
-Some Microsoft and third-party platforms don't support .NET 5. For example, some Azure services provide an SDK that isn't yet available for consumption on .NET 5. Most Azure SDK should eventually be ported to .NET 5/Standard but some might not for various reasons. You can see the available Azure SDKs in the [Azure SDK Latest Releases](https://azure.github.io/azure-sdk/releases/latest/index.html) page.
+Some Microsoft and third-party platforms don't support .NET 6. For example, some Azure services provide an SDK that isn't yet available for consumption on .NET 6. Most Azure SDK should eventually be ported to .NET 6/Standard but some might not for various reasons. You can see the available Azure SDKs in the [Azure SDK Latest Releases](https://azure.github.io/azure-sdk/releases/latest/index.html) page.
```
Comment (nishanil) [inline docs/architecture/microservices/net-core-net-framework-containers/net-framework-container-scenarios.md:40]:
```suggestion
Some Microsoft and third-party platforms don't support .NET 6. For example, some Azure services provide an SDK that isn't yet available for consumption on .NET 6 yet. Most Azure SDK should eventually be ported to .NET 6/.NET Standard, but some might not for several reasons. You can see the available Azure SDKs in the [Azure SDK Latest Releases](https://azure.github.io/azure-sdk/releases/latest/index.html) page.
```

File: docs/architecture/microservices/net-core-net-framework-containers/official-net-docker-images.md [hunk 8]
```diff
@@ -42,8 +42,8 @@ When you explore the .NET image repositories at Docker Hub, you will find multip
 
 | Image | Comments |
 |-------|----------|
-| mcr.microsoft.com/dotnet/aspnet:**5.0** | ASP.NET Core, with runtime only and ASP.NET Core optimizations, on Linux and Windows (multi-arch) |
-| mcr.microsoft.com/dotnet/sdk:**5.0** | .NET 5, with SDKs included, on Linux and Windows (multi-arch) |
+| mcr.microsoft.com/dotnet/aspnet:**6.0** | ASP.NET Core, with runtime only and ASP.NET Core optimizations, on Linux and Windows (multi-arch) |
+| mcr.microsoft.com/dotnet/sdk:**6.0** | .NET 6, with SDKs included, on Linux and Windows (multi-arch) |
```
Comment (nishanil) [inline docs/architecture/microservices/net-core-net-framework-containers/official-net-docker-images.md:46]: Below this table, can you link to this page: https://github.com/dotnet/dotnet-docker ? Also, mention the avilability of Nightly builds too.

File: docs/architecture/microservices/architect-microservice-container-applications/scalable-available-multi-container-microservice-applications.md [hunk 9]
```diff
@@ -33,6 +33,7 @@ The concepts of a cluster and a scheduler are closely related, so the products p
...
 | **Azure Kubernetes Service (AKS)** <br> ![An image of the Azure Kubernetes Service logo.](./media/scalable-available-multi-container-microservice-applications/azure-kubernetes-service-logo.png) | [AKS](https://azure.microsoft.com/services/kubernetes-service/) is a managed Kubernetes container orchestration service in Azure that simplifies Kubernetes cluster's management, deployment, and operations. |
+| **Azure Container Apps** <br> ![An image of the Azure Container Apps Service logo.](./media/scalable-available-multi-container-microservice-applications/azure-container-apps-logo.png) | [Azure Container Apps](https://azure.microsoft.com/services/container-apps/) is a managed serverless container service for building and deploying modern apps at scale. |
```
Comment (nishanil) [inline docs/architecture/microservices/architect-microservice-container-applications/scalable-available-multi-container-microservice-applications.md:36]: Container Apps logo is too big. May be reduce to fit the table? Check the review site for details.

**Review by nishanil — COMMENTED** [review]
Some more commments..

File: docs/architecture/microservices/net-core-net-framework-containers/net-framework-container-scenarios.md (hunk 6 above)
Comment (sughosneo) [reply docs/architecture/microservices/net-core-net-framework-containers/net-framework-container-scenarios.md]: Yes, correct. Removed the links.

File: docs/architecture/microservices/docker-application-development-process/docker-app-development-workflow.md (hunk 5 above)
Comment (sughosneo) [reply docs/architecture/microservices/docker-application-development-process/docker-app-development-workflow.md:50]: Yes, changed it.

File: docs/architecture/microservices/net-core-net-framework-containers/official-net-docker-images.md (hunk 8 above)
Comment (sughosneo) [reply docs/architecture/microservices/net-core-net-framework-containers/official-net-docker-images.md:46]: sure, mentioned in after the table.

File: docs/architecture/microservices/architect-microservice-container-applications/scalable-available-multi-container-microservice-applications.md (hunk 9 above)
Comment (sughosneo) [reply docs/architecture/microservices/architect-microservice-container-applications/scalable-available-multi-container-microservice-applications.md:36]: Now, updated the container apps logo to fit the table.

**Comment by sughosneo** [comment]
> @sughosneo Looks good. Some minor things to be taken care of.

Thank you @nishanil for your review. I have included all the changes.

**Review by nishanil — APPROVED** [review]
LGTM! Thanks @sughosneo 
@IEvangelist Feel free to merge this PR, when you are ready.

**Review by IEvangelist — APPROVED** [review]

**Pull request merged**